source/docs/03-metadata.xml

```diff
@@ -1307,7 +1307,7 @@
                      <specDesc key="extMeta"/>
                   </specList>
                </p>
-               <p>The content of the <gi scheme="MEI">item</gi> is quite similar to <gi scheme="MEI">manifestation</gi> with a few omissions:</p>
+               <p>The content of the <gi scheme="MEI">item</gi> element is quite similar to the <gi scheme="MEI">manifestation</gi> element. The <gi scheme="MEI">item</gi> level is used to describe specifics concerning a single item. This information can differ from the description at the <gi scheme="MEI">manifestation</gi> level or can be additional information. The following elements may be used:</p>
```

Comment:
I have removed the markup for the manifestation *element* because the text seems to talk about the manifestation *level*, and I don't think we should point to the element if the text is discussing the FRBR concept of manifestation.

If this is not what is intended, please let me know.

Reply:
Thank you! I totally agree.